mesonbuild/compilers/compilers.py

```diff
@@ -119,6 +119,13 @@ def is_library(fname):
                        'minsize': ["/MD", "/Zi", "/Os", "/Ob1"],
                        }
 
+watcom_buildtype_args = {'plain': [],
+                         'debug': ["-ad", "-od", "-d2"],
+                         'debugoptimized': ["-ad", "-onatx", "-d2"],
+                         'release': ["-ad", "-onatx"],
+                         'minsize': ["-ad", "-onasx"],
+                        }
```

Comment:
__[flake8]__
* [E124] closing bracket does not match visual indentation

<!-- Comment by [SideCI](https://sideci.com) -->